cogs/utility.py

```diff
@@ -21,7 +21,7 @@
 
 import datetime
 from time import monotonic
-from typing import List
+from typing import Union
```

Comment:
may I ask why this is being imported?

Reply:
I had planned to use Union to allow for multiple versions of a variable, typing,Union tries to turn the argument into different types, I can do Union[discord.Member, discord.User, int], and then it'll try and turn it into a member, if that fails, turn into User, if that fails turn into an int which I can then turn into a user by fetching the user. I then realized we don't need anything from any of those types, and int would be just fine. I then forgot to remove the import.

Reply:
Yeah I know what Union is, I've been using it as well for my bot. And alright. Everything seems to be fine except line 347. You forgot to add `name=` as a positional argument for embed.add_field()

Reply:
Oh yup, just fixed both issues!